packages/jellyfish-testing/src/testing.ts

```diff
@@ -6,24 +6,34 @@ import { TestingRawTx } from './rawtx'
 import { TestingICX } from './icxorderbook'
 import { TestingMisc } from './misc'
 import { TestingGroupAnchor } from './anchor'
-import { ContainerGroup, MasterNodeRegTestContainer, StartOptions } from '@defichain/testcontainers'
+import { ContainerGroup, DeFiDContainer, MasterNodeRegTestContainer, StartOptions } from '@defichain/testcontainers'
 import { RegTestFoundationKeys } from '@defichain/jellyfish-network'
 import { JsonRpcClient } from '@defichain/jellyfish-api-jsonrpc'
 
 export class Testing {
-  public readonly fixture = new TestingFixture(this)
-  public readonly token = new TestingToken(this.container, this.rpc)
-  public readonly poolpair = new TestingPoolPair(this.container, this.rpc)
-  public readonly rawtx = new TestingRawTx(this.container, this.rpc)
-  public readonly icxorderbook = new TestingICX(this)
-  public readonly misc = new TestingMisc(this.container, this.rpc)
+  public readonly container: MasterNodeRegTestContainer
```

Comment:
Suggestion

Add a parent class for Testing, which allow the `container` to be `RegTestContainer`
Thus when `Testing` standalone must use MNRegTest, `TestingGroup` can have `testings: RegTestTesting[]`

Reply:
initial attempts on this will result in needing a way to distinguish the returning of `Testing` and `NonMNTesting` it could end by needing to use `as` again. not good in typescript to think of any tricks here